contracts/token/ERC20/ERC20.sol

```diff
@@ -140,7 +140,7 @@ contract ERC20 is IERC20 {
      * @param value The amount that will be created.
      */
     function _mint(address account, uint256 value) internal {
-        require(account != address(0));
+        require(account != address(0), "empty or zero transaction recipient");
```

Comment:
Similarly, this should be `mint`, and the others `burn`, `approve`, etc. No two reason strings should read the same.

Reply:
yes correct.